MovingBlocks/Terasology
# Fix UI for new screens

### Contains

According to this comment from @Cervator https://github.com/MovingBlocks/Terasology/issues/3400#issuecomment-399691891, I've added second row for back button and play buttons. 
If @TheFlash98 or @iaronaraujo are doing the same at the moment, please let me know. 

### How to test

Select Game -> Create -> Advanced -> (next screens)
Check that all Play buttons work properly and screen views have been improved.

### Outstanding before merging

Unfortunately, some of new screens are still under development. However, I was able to fix
some views. And I'm going to fix other when they will be ready in separate PR. 

- [x] "Play" buttons aren't active yet.
